node/node.go

```diff
@@ -617,7 +617,7 @@ func (n *Node) createConsensusTopic(messageProcessor p2p.MessageProcessor) error
 
 	n.consensusTopic = core.ConsensusTopic + n.shardCoordinator.CommunicationIdentifier(n.shardCoordinator.SelfId())
 	if n.messenger.HasTopicValidator(n.consensusTopic) {
-		return ErrValidatorAlreadySet
+		return n.messenger.RegisterMessageProcessor(n.consensusTopic, messageProcessor)
```

Comment:
this should be reverted. The check is done on the HasTopicValidator, not HasTopic